src/pages/Overview/BalanceLinks.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 @paritytech/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: GPL-3.0-only
+
+import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
+import { ButtonPrimaryInvert, Separator } from '@polkadot-cloud/react';
+import { useTranslation } from 'react-i18next';
+import { useApi } from 'contexts/Api';
+import { useConnect } from 'contexts/Connect';
+import { MoreWrapper } from './Wrappers';
+
+export const BalanceLinks = () => {
+  const { t } = useTranslation('pages');
+  const { name } = useApi().network;
+  const { activeAccount } = useConnect();
+
+  return (
+    <MoreWrapper>
+      <Separator />
+      <h4>{t('overview.moreResources')}</h4>
+      <section>
+        <ButtonPrimaryInvert
```

Comment:
I've disabled it on all networks, but mainnet, because that's the only one where we have subscan. On mainnet the button works and redirects to `https://alephzero.subscan.io/account/{accountId}`